packages/types-cloudflare-worker/src/global.ts

```diff
@@ -293,14 +294,164 @@ export interface CloudflareRequestAttributes extends CloudflareRequestFeatures {
    *
    * Business and Enterprise ONLY.
    */
-  readonly regionCode?: string;
+  readonly regionCode: string;
 
   /**
    * Timezone of the incoming request. (e.g. "America/Chicago")
    *
    * Business and Enterprise ONLY.
    */
-  readonly timezone?: string;
+  readonly timezone: string;
+}
+
+/**
+ * Cloudflare Request Attributes Mock
+ *
+ * Same fields as CloudflareRequestAttributes, but optional and not read-only.
+ * Warning: Must be kept in sync with CloudflareRequestAttributes.
+ */
+export interface MockCloudflareRequestAttributes
```

Comment:
It looks like the [`Partial`](https://www.typescriptlang.org/docs/handbook/utility-types.html#partialt) utility could be really helpful here! 😄

Reply:
:rocket: Perfect!